src/main/java/mat/server/service/FhirValidationReportService.java

```diff
@@ -95,31 +101,79 @@ private String generateValidationReport(ConversionResultDto conversionResultDto,
         paramsMap.put("matVersion", currentMatVersion);
 
         if (conversionResultDto != null && measure != null) {
-            //Measure FHIR validation errors
-            List<FhirValidationResult> measureFhirValidationErrors = conversionResultDto.
-                    getMeasureConversionResults().getMeasureFhirValidationResults();
-
-            // Library FHIR validation errors
-            List<LibraryConversionResults> libraryConversionResults = conversionResultDto.getLibraryConversionResults();
-            List<FhirValidationResult> libraryFhirValidationErrors = libraryConversionResults.stream().flatMap(i -> i.getLibraryFhirValidationResults().stream()).collect(Collectors.toList());
-
-            // CQL conversion errors
-            List<CqlConversionError> cqlConversionErrors = libraryConversionResults.stream().map(i -> i.getCqlConversionResult()).flatMap(i -> i.getCqlConversionErrors().stream()).collect(Collectors.toList());
-
-            // ValueSet FHIR validation errors
-            List<FhirValidationResult> valueSetFhirValidationErrors = conversionResultDto.getValueSetConversionResults().stream().flatMap(i -> i.getValueSetFhirValidationResults().stream()).collect(Collectors.toList());
+            if (!("SUCCESS").equals(conversionResultDto.getOutcome().name())) {
```

Comment:
What was the reason you didn't use enum literals?

if (ConversionOutcome.SUCCESS != conversionResultDto.getOutcome())

Reply:
No specific reason maybe ignorance. 
I will update that.